Makefile

```diff
@@ -8,9 +8,9 @@
 # Install ineffassign with: go install github.com/gordonklaus/ineffassign@latest
 # Install golint with:      go install golang.org/x/lint/golint@latest
 
-all: lint maypanic statistics reachability dependencies static-commands render taint compare defer packagescan backtrace argot-cli racerg setup-precommit
+all: setup-precommit lint racerg argot-build test
```

Comment:
nit: I'd like to just have to type `make argot` or `make build` instead of `make argot-build`